core/src/i18n/en.yaml

```diff
@@ -1,40 +1,89 @@
+---
+addons:
+  title: Addons
+api-rules:
+  title: API Rules
+applications:
+  title: Applications
+brokers:
+  title: Brokers
+catalog:
+  menu-title: Catalog
+  subtitle: Enrich your experience with additional services that are installed outside the cluster.
+  title: Service Catalog
+cluster-addons:
+  title: Cluster Addons
+cluster-role-bindings:
+  title: Cluster Role Bindings
+cluster-roles:
+  title: Cluster Roles
 clusters:
   overview:
     title: Cluster Overview
+config-maps:
+  title: Config Maps
+configuration:
+  title: Configuration
+cron-jobs:
+  concurrency-policy:
+    descriptions:
+      allow: Allow concurrent executions.
+      replace: No concurrent executions, replace previous run with the new one.
+      skip: No concurrent executions, skip new run if the previous one hasn't finished yet.
+    title: Concurrency Policy
+  last-job-execution: Last job execution
+  last-schedule-time: Last schedule time
+  not-scheduled-yet: None scheduled yet
+  schedule: Schedule
+  title: Cron Jobs
+custom-resource-definitions:
+  title: Custom Resource Definitions
+deployments:
+  title: Deployments
+discovery-and-network:
+  title: Discovery and Network
+functions:
+  title: Functions
+git-repositories:
+  title: Git Repositories
+instances:
+  title: Instances
+integration:
+  title: Integration
 namespaces:
   overview:
+    back: Back to Namespaces
+    resources:
+      error: Error while loading memory consumption data
+      limits: Memory limits
+      requests: Memory requests
+      title: Resource consumption
     title: Overview
     workloads:
-      title: Healthy Resources
-      pods: $t(pods.title)
       deployments: $t(deployments.title)
       error: 'Error while loading {{title}} data due to: {{message}}'
-    resources:
-      title: Resource consumption
-      requests: Memory requests
-      limits: Memory limits
-      error: Error while loading memory consumption data
+      pods: $t(pods.title)
+      title: Healthy Resources
+  title: Namespaces
+oauth2-clients:
+  title: OAuth2 Clients
+observability:
+  title: Observability
 pods:
   title: Pods
-deployments:
-  title: Deployments
-service-catalog:
+replica-sets:
+  title: Replica Sets
+role-bindings:
+  title: Role Bindings
+roles:
+  title: Roles
+secrets:
+  title: Secrets
+service-management:
```

Comment:
Maybe we could split those into categories as well?

something like:
```
service-management:
  title: Service Management
  children:
  - catalog:
    title: Service Catalog
  - ...
```
or 
```
categories:
 - service-management: ...
subcategories:
- catalog: ...
```
Just ideas to discuss, not suggestions. It just confuses me that 2 different things are on the same level.